Test/Core/Math/MathFTest.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Text;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+using Duality;
+
+using NUnit.Framework;
+
+namespace Duality.Tests.Utility
+{
+	[TestFixture]
+	public class MathFTest
+	{
+		private const float Epsilon = 0.0001f;
+
+		[Test] public void Clamp01Checks()
+		{
+			Assert.AreEqual(MathF.Clamp01(-.0f), 0.0f, Epsilon);
+			Assert.AreEqual(MathF.Clamp01(.0f), 0.0f, Epsilon);
+			Assert.AreEqual(MathF.Clamp01(.001f), 0.001f, Epsilon);
+			Assert.AreEqual(MathF.Clamp01(1.0001f), 1.0f, Epsilon);
+			Assert.AreEqual(MathF.Clamp01(1.0f), 1.0f, Epsilon);
+			Assert.AreEqual(MathF.Clamp01(.999f), .999f, Epsilon);
+			Assert.AreEqual(MathF.Clamp01(.567f), .567f, Epsilon);
```

Comment:
Code Style: For float literals, always write out both before and after the separator dot, e.g. `0.1f` instead of `.1f` and `1.0f` instead of `1f`.